geemdjem0819/Assets/scripts/mechanics/SunCircleController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace mechanics
+{
+    public class SunCircleController : MonoBehaviour
+    {
+        public float TimeOfday
+        {
+            get => Mathf.Clamp(_timeOfday, 0, 365);

```

Comment:
Is het per sé nuttig om de volledige nacht mee te laten doen? dusk/dawn is in principe wat logischer misschien? wat doen planten eigenlijk 's nachts? slapen ze? T is iig geen fotosynthesetijd... toch? tenzij je een lamp hebt, dan...